packages/legacy/core/App/hooks/notifications.ts

```diff
@@ -64,5 +74,11 @@ export const useNotifications = (): Array<BasicMessageRecord | CredentialRecord
     setNotifications(notif)
   }, [basicMessages, credsReceived, proofsDone, proofsRequested, offers, credsDone])
 
+  useEffect(()=> {
+    if(openIDCredsRecieved) {
+      setNotifications([openIDCredsRecieved])
+    } 
+  }, [openIDCredsRecieved])
```

Comment:
This will clear all other notifications after the user scans an OpenID offer, no?